nixos/modules/services/security/oauth2-proxy.nix

```diff
@@ -547,35 +545,37 @@ in
         OAUTH2_PROXY_CLIENT_SECRET=asdfasdfasdf.apps.googleuserscontent.com
         and specify the path here.
       '';
-      example = "/run/keys/oauth2_proxy";
+      example = "/run/keys/oauth2-proxy";
     };
-
   };
 
-  config = mkIf cfg.enable {
+  imports = [
+    (mkRenamedOptionModule [ "services" "oauth2_proxy" ] [ "services" "oauth2-proxy" ])
+  ];
```

Comment:
nonblocking:

It doesn't matter much, and I can't recall seeing any guidelines specifying the order of modules, but it feels a bit weird to put `imports` between `options` and `config`.

While looking through, I only found `nginx` that does it like this, and `postfix` which puts the imports on the bottom.

Reply:
Languages like C and Python put includes and imports are at the top.

Reply:
This is pretty standard across the repo